aliyun_img_utils/aliyun_image.py

```diff
@@ -146,6 +165,178 @@ def upload_image_tarball(
 
         return blob_name
 
+    def delete_compute_image(self, image_name, delete_blob=False):
+        """
+        Delete compute image in current region.
+
+        If delete_blob is True delete the backing storage blob.
+        """
+        try:
+            image = self.get_compute_image(image_name=image_name)
+        except AliyunImageException:
+            return False
+
+        request = DeleteImageRequest()
+        request.set_accept_format('json')
+        request.set_ImageId(image['ImageId'])
+
+        try:
+            self.compute_client.do_action_with_exception(request)
+        except Exception as error:
+            raise AliyunImageException(
+                f'Unable to delete image: {error}.'
+            )
+
+        self.wait_on_compute_image_delete(image['ImageId'])
+
+        if delete_blob:
+            device = image['DiskDeviceMappings']['DiskDeviceMapping'][0]
+            self.delete_image_tarball(device['ImportOSSObject'])
```

Comment:
In alibaba the image is not in a tarball, what gets uploaded is a file in qcow2 format. I suggest we call this `delete_image_file` or `delete_image_backing_store`

Reply:
Agree, I actually think the method is better named delete_storage_blob since it's not really specific to images.